hw3/01-characters.js

```diff
@@ -1,4 +1,55 @@
 // url for the Thrones API
-const url = 'https://thronesapi.com/api/v2/Characters';
+const url = "https://thronesapi.com/api/v2/Characters";
 
-fetch(url);
+//reference --https://www.javascripttutorial.net/javascript-fetch-api/
+
+let section = document.querySelector("section");
+
+function show(character) {
+  let div = document.createElement("div");
+  let image = document.createElement("img");
+  let name = document.createElement("h2");
+  let title = document.createElement("p");
```

Comment:
Consider using the figure and figcaption elements, this exercise would be a good use case for them.